src/main/java/com/testmonitor/actions/Milestones.java

```diff
@@ -127,18 +129,35 @@ public Milestone create(Milestone milestone) throws IOException {
     /**
      * Find a milestone using the provided query or create a new one.
```

Comment:
Find a milestone using the provided query or create a new one.

- Using the provided name?
- Does it create a new one?